### ISSUE TITLE
bug: `websocket.addEventListener` not properly wrapped

### ISSUE THREAD

**Issue by eugeneware** [angular/zone.js#35]
Even though #9 shows that websockets is working it does not seem to be the case.

For example when messages get received from a websocket, the `afterTask` doesn't run:

``` js
function run() {
  var sock = new WebSocket('ws://' + location.host + '/changes/websocket');
  sock.addEventListener('open', function (evt) {
    sock.send(JSON.stringify({ hello: 'world' }));
  });

  sock.addEventListener('message', function (evt) {
    var msg = JSON.parse(evt.data);
    console.log('message', msg);
  });
}

zone.fork(watchZone()).run(run);
function watchZone() {
  return {
    afterTask: function () {
      console.log('after');
    }
  };
}
```

**Comment by btford** [comment]
See also https://github.com/angular/angular/issues/487#event-227266387

**Comment by btford** [comment]
This should be fixed in v0.4.0.
